src/NuGet.Clients/NuGet.SolutionRestoreManager/VsSolutionRestoreService.cs

```diff
@@ -124,30 +115,55 @@ public Task<bool> NominateProjectAsync(string projectUniqueName, IVsProjectResto
             }
         }
 
-        private void DumpProjectRestoreInfo(PackageSpec packageSpec)
+        private void DumpProjectRestoreInfo(DependencyGraphSpec projectRestoreInfo)
```

Comment:
Could this whole method be put in `#if DEBUG`? It is dead code when building Release.